broker-core/src/main/java/io/zeebe/broker/workflow/processor/event/TriggerBoundaryEventHandler.java

```diff
@@ -48,6 +48,9 @@ public void handle(BpmnStepContext<ExecutableBoundaryEvent> context) {
       context
```

Comment:
Can we extract the branches to methods like `handleNonInterruptingBoundaryEvent` and `handleInterruptingBoundaryEvent`